pages/download/preview.html

```diff
@@ -141,7 +141,7 @@ <h2>Need a stable version?</h2>
 		<p>Want to stick to the <strong>true and trusted Godot 3</strong>? Download the long-term support version!</p>
```

Comment:
Should this line change too?

Reply:
Yes! Forgot to check about the text format of LTS.